README.md

```diff
@@ -31,14 +33,20 @@ bundle exec rake db:create db:migrate
 bundle exec rails server
 ```
 
+# Test client application
+
+You can find example of Barong usage here: [Barong Test Client App](https://github.com/rubykube/barong-client-app)
+
+[How to get client app credentials](./docs/doorkeeper.md)
```

Comment:
rename this into oauthclient.md
and also add the application seed information with the skipauth true/false flag.